src/main/kotlin/no/nav/familie/klage/behandling/domain/Klagebehandling.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.familie.klage.behandling.domain
+
+import no.nav.familie.kontrakter.felles.klage.BehandlingResultat
+import no.nav.familie.kontrakter.felles.klage.BehandlingStatus
+import no.nav.familie.kontrakter.felles.klage.KlagebehandlingDto
+import no.nav.familie.kontrakter.felles.klage.Årsak
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class Klagebehandling(
```

Comment:
Tenker vi. åbruke denne internt eller eksternt?

Reply:
Internt, blir brukt for å hente opp relevant data fra databasen. Blir mappet til KlageBehandlingDto i kontrakter som hentes fra ef-sak